src/components/TradeWidget/index.tsx

```diff
@@ -217,10 +226,6 @@ const TradeWidget: React.FC = () => {
         },
       })
       if (success && pendingTxHash) {
-        // reset form on successful order placing
-        reset(DEFAULT_FORM_STATE)
-        setUnlimited(false)
-        // remove pending tx
```

Comment:
What I meant in the previous comment was: do you need push this test?
In other words, will the form no longer reset and `setUnlimit false` ?

Reply:
it shouldnt - if a user is in the middle of making a new order and the old tx mines, then the form resets... sucks as UX

Reply:
yeah probably best to move this directly after tx hash, not remove entirely